docs/docs/concepts/01-intro.md

```diff
@@ -36,13 +36,13 @@ Visit [our website](https://aica.tech) or [contact us](mailto:contact@aica.tech)
 offerings.
 :::
 
-## AICA Developer Interface
+## AICA Studio
 
-The **Developer Interface** is the user interface layer to the AICA framework that empowers developers to build and
+**AICA Studio** is the user interface layer to AICA Core that empowers developers to build and
 extend their own advanced robotic applications. AICA developers use this interface internally when building custom
 applications, and decided it was worth sharing with the robotics community as a stand-alone product with the following
 features:
 
 - Graphical web-based UI to dynamically edit, run and monitor AICA application
 - REST API to interface with and manage AICA applications from CLI or automation cells
-- Component SDK to extend the core library with custom logic and behaviors
+- Component SDK to extend the base library with custom logic and behaviors
```

Comment:
```suggestion
- Component SDK to extend the base library with custom logic and behaviors
```
Should we say `libraries` ? Idk

Reply:
There is only one library, no? Or is base split into multiple libraries?

Reply:
there is one "library of packages", but for that I usually use the word "registry" instead. Then you can think of "software libraries" as many individual modules as part of one or more packages. 

In this context I still prefer the singular library.